manifests/Sheragi Rebirth.yaml

```diff
@@ -0,0 +1,13 @@
+name: Sheragi Rebirth

```

Comment:
```suggestion
name: Sheragi Rebirth
```

Reply:
Hi, I'm new to using github, so any help you provide would be appreciated. The suggested change you've made doesn't appear to be any different, so if you could please elaborate on it, thank you.

Reply:
Oh yeah GitHub seems to have messed that up. The line ending at the end is wrong.

Reply:
wrong in what way, and if you wouldn't mind helping me fix it.